### DISCUSSION TITLE
v4.4.5 missing on App Store?

### DISCUSSION THREAD

**Discussion by brentalanmiller** [utmapp/UTM#6032] [General]
It's been a while since v4.4.5 has been released, but it looks like the MacOS App Store is still on v.4.4.4. Was there an issue getting v4.4.5 into the store?

**Comment by osy** [comment]
Read the release notes for v4.4.5

**Comment by brentalanmiller** [comment]
Apologies, but where's that? I didn't see anything on https://github.com/utmapp/UTM/releases/tag/v4.4.5 nor https://docs.getutm.app/updates/v4.4/

**Reply by osy** [reply]
v4.4.5 only has changes for iOS specifically to support TrollStore 2. The macOS version is the same build as v4.4.4.

**Reply by brentalanmiller** [reply]
Oh, gotcha. Thanks for the clarification. I misread the notes and thought there were improvements for Apple Virtualization in there if we were running macOS Sonoma.
